lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -0,0 +1,90 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "rubocop"
+rescue LoadError
+  return
+end
+
+require "cgi"
+require "singleton"
+
+module RubyLsp
+  module Requests
+    module Support
+      # :nodoc:
+      class RuboCopRunner < RuboCop::Runner
+        extend T::Sig
+
+        sig { returns(T::Array[RuboCop::Cop::Offense]) }
+        attr_reader :offenses
+
+        DEFAULT_ARGS = T.let([
+          "--force-exclusion",
+        ].freeze, T::Array[String])
+
+        sig { params(args: String).void }
+        def initialize(*args)
+          @options = T.let({}, T::Hash[Symbol, T.untyped])
+          @offenses = T.let([], T::Array[RuboCop::Cop::Offense])
+          @errors = T.let([], T::Array[String])
+          @warnings = T.let([], T::Array[String])
+
+          args += DEFAULT_ARGS
+          rubocop_options = ::RuboCop::Options.new.parse(args).first
+          super(rubocop_options, ::RuboCop::ConfigStore.new)
+        end
+
+        sig { params(path: String, contents: String).void }
+        def run(path, contents)
+          @errors = []
+          @warnings = []
+          @offenses = []
+          @options[:stdin] = contents
+          capture_output { super([path]) }
+          display_handled_errors
+        end
+
+        sig { returns(String) }
+        def formatted_source
+          @options[:stdin]
+        end
+
+        private
+
+        sig { void }
+        def display_handled_errors
```

Comment:
This is so that people can see and/or report problems that any formatting/diagnostic operation has encountered. I feel like this is needed more for troubleshooting than making people aware. After all, there is no action to be taken on any of these, since they are already handled.

This is what the output looks like:
```
[RuboCop] Encountered and handled errors:
[RuboCop]   - An error occurred while Layout/BlockEndNewline cop was inspecting /Users/ufuk/src/github.com/Shopify/tapioca/lib/tapioca/static/rbs/parameter_converter.rb:4:17.
```